vendor/assets/javascrips/fileupload/jquery.fileupload.js

```diff
@@ -0,0 +1,922 @@
+/*
```

Comment:
This doesn't seem like the current version of jquery.fileupload.js: https://github.com/blueimp/jQuery-File-Upload/blob/eb985ea473e021ffe714096e0f03f5a3f56048d4/js/jquery.fileupload.js